crates/edr_rpc_hardhat/src/lib.rs

```diff
@@ -118,7 +119,7 @@ pub enum Request {
         serialize_with = "single_to_sequence",
         deserialize_with = "sequence_to_single"
     )]
-    SetPrevRandao(ZeroXPrefixedBytes),
+    SetPrevRandao(B256),
```

Comment:
Just double checking: does `B256` serializes into `0x...` string?

Reply:
Yes, it does.